tools/DynamoPackagesAnalyzer/DynamoPackagesAnalyzer.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.5.33627.172
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DynamoPackagesAnalyzer", "DynamoPackagesAnalyzer.csproj", "{1A49CE82-84F0-479E-9A85-7A7437D12A02}"
```

Comment:
does this .sln serve a purpose since it only has one .csproj?

Reply:
It may work without a solution file, but almost all of the tools at /tools folder have the *.sln file, and visual studio will try to create a new solution file each time, thats the reason I wanted to keep the sln file